tests/test_startup_warmup.py

```
@@ -107,6 +108,98 @@ async def slow_boom() -> None:
+    monkeypatch.setattr(server, "ensure_bootstrap", _slow_bootstrap)
+    monkeypatch.setattr(server, "_warmup_step", _warmup)
+    monkeypatch.setattr(
+        "services.background.start_background_services", _start_background
+    )
+    monkeypatch.setattr("services.background.run_background_in_web", lambda: True)
+
+    cm = server.lifespan(server.app)
+    try:
+        await cm.__aenter__()
+    except Exception:
+        pass  # the lifespan is halted deliberately once ordering is observed
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)